release notes/v0.36.0.md

```diff
@@ -0,0 +1,108 @@
+k6 v0.36.0 is here! 🎉 It introduces a couple new features which enhance its usability, and includes a bunch of fixes and the result of refactoring ongoing efforts.
+
+In total, we have closed 27 issues.
+
+## New Features!
+
+### Sourcemaps support ([#2082](https://github.com/grafana/k6/pull/2082))
+
+Following [#2082](https://github.com/grafana/k6/pull/2082), k6's JS interpreter now has support for [Source Maps](https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Use_a_source_map). In the context of k6 scripts, it ensures that exceptions thrown from imported modules are reported with correct line information; as is described in [#1804](https://github.com/grafana/k6/issues/1804).
+
+Thus, given an `imported.js` module such as:
+```javascript
+export function f1() {
+  throw "line 2";
+}
+
+export function f2() {
+  throw "line 6";
+}
+
+export function f3() {
+  throw "line 10";
+}
+```
+
+and a k6 test script importing it as such:
+```javascript
+import { f2 } from "./imported.js"
+
+export default function() {
+  f2();
+}
+```
+
+k6 would indeed report the exception at the correct line in `imported.js`, with an output along the lines of:
+```
+ERRO[0000] line 6
+        at f2 (file:///some/path/imported.js:6:2(2))
+        at file:///some/path/loadtest.js:4:2(4)
+        at native  executor=per-vu-iterations scenario=default source=stacktrace
+```
```

Comment:
maybe also show the previous result?

Reply:
I see that you resolved this and :+1: but I don't see where you show the previous result, @oleiade

Reply:
I have refactored this section to illustrate what the output would have been in previous versions, and comparing it to `v0.36.0`'s output. Let me know if that fits what you expected 🤞🏻